client/src/CasesImport/CasesImport.js

```diff
@@ -14,48 +13,62 @@ export function CasesImport() {
 
   const columns = getColumns(t)
 
+  const getType = file => {
+    const nameArray = file.name.split('.')
+    const extension = nameArray[nameArray.length - 1]
+    // sometimes xls files come in with an empty type so we want to parse
+    // the extension to verify what parser to use
+    if (file.type === '' && (extension === 'xls' || 'xlsx')) {
```

Comment:
I think you meant to write `extension === 'xls' || extension === 'xlsx`.

We could also write it as `['xls', 'xlsx'].includes(extension)`.

Reply:
I sure did mean to write that.  Will fix.